### ISSUE TITLE
prefer-explicit-string-conversion Recommend String(x) over `${x}`

### ISSUE THREAD

**Issue by graingert** [eslint/eslint#9737]
**Please describe what the rule should do:**
Recommend String(x) over \`${x}\` or `'' + x`
**What category of rule is this? (place an "X" next to just one item)**

[ ] Enforces code style
[ ] Warns about a potential error
[x] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js
// bad:
const foo = `${x}`;
// bad:
const foo = '' + x;
// good:
const foo = String(x);
```

**Why should this rule be included in ESLint (instead of a plugin)?**

Pretty simple, no stage-3 or lower features.

**Comment by j-f1** [comment]
This looks like it’s covered by the [`string`](https://eslint.org/docs/rules/no-implicit-coercion#string) option of [`no-implicit-coercion`](https://eslint.org/docs/rules/no-implicit-coercion).

**Comment by platinumazure** [comment]
Closing as it looks like this question has been answered. Please feel free to comment here or to stop by our [Gitter chat](https://gitter.im/eslint/eslint) if you need any additional help. Thanks!
